src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -2,10 +2,14 @@
 import threading
 from typing import List, Union
 from ipykernel.comm import Comm
+from biokbase.narrative.jobs.util import load_job_constants
 from biokbase.narrative.jobs.jobmanager import JobManager
 from biokbase.narrative.exception_util import NarrativeException, JobRequestException
 from biokbase.narrative.common import kblogging
 
+
+(PARAM, MESSAGE_TYPE) = load_job_constants()
```

Comment:
the remainder of the changes in this file are from switching out constants like `JOB_ID` or `CANCEL` for `PARAM["JOB_ID"]` or `MESSAGE_TYPE["CANCEL"]`